src/lib/storage/fixed_width_attribute_vector.cpp

```diff
@@ -0,0 +1,38 @@
+#include "fixed_width_attribute_vector.hpp"
+#include "all_type_variant.hpp"
+namespace opossum {
+
+template <typename T>
+FixedWidthAttributeVector<T>::FixedWidthAttributeVector(const size_t size) {
+  _values.reserve(size);
+}
+
+template <typename T>
+ValueID FixedWidthAttributeVector<T>::get(const size_t index) const {
+  return static_cast<ValueID>(_values.at(index));
+}
+
+template <typename T>
+void FixedWidthAttributeVector<T>::set(const size_t index, const ValueID value_id) {
+  if (_values.size() <= index) {
+    _values.push_back(value_id);
+  } else {
```

Comment:
This case should not occur because the AttributeVector is already constructed with the right size.
If this happens nevertheless, setting the item of position 20 in an AttributeVector with size 10, the item would be pushed to position 10.

Reply:
Good point!